web/client/epics/geostory.js

```diff
@@ -208,6 +263,17 @@ export const loadGeostoryEpic = (action$, {getState = () => {}}) => action$
                 }
             ))
         );
+/**
+ * Triggers reload of last loaded story when user login-logout
+ * @param {Observable} action$ the stream of redux actions
+ */
+export const reloadGeoStoryOnLoginLogout = (action$) =>
```

Comment:
you do the reload for permissions checks?
and for rendering feedback mask ?

Reply:
The story reload should trigger automatically the permission and feedback mask actions (so in case of missing permission, show do the correct actions to show the feedkback mask, prompt the login and so on). 

Did you tried it and you found it doesn't work in some case? If so, please notify me the bug. I'll fix it